identity-account/src/stronghold/records.rs

```diff
@@ -48,18 +50,23 @@ impl Records<'_> {
   }
 
   pub async fn index(&self) -> Result<RecordIndex> {
```

Comment:
Since we're keeping this now, the methods here should return `IotaStrongholdResult`.